topi/python/topi/x86/conv2d.py

```diff
@@ -415,11 +415,15 @@ def _alter_conv2d_layout(attrs, inputs, tinfo, F):
 
     dtype = data.dtype
     out_dtype = dtype if out_dtype in ("same", "") else out_dtype
-    is_depthwise = groups == in_channel and groups == out_channel
 
     # only optimize for NCHW
     if layout != 'NCHW':
```

Comment:
```suggestion
    if layout != 'NCHW' or attrs["kernel_layout"] != "OIHW":
```
and remove the assert below.